### PR TITLE
NINJA-193 create groups via api

### PR DESCRIPTION
**Pull request by ParryLuke**

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -1005,6 +1005,31 @@ paths:
...
+        '422':
+          description: |
+            The group was not saved due to Drupal validation failure or JSON schema validation
+            failure.
+      description: Create a group.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/group-schema'
```
Comment (jmorahan) [inline api.swagger.yaml]: Some fields should be required on create

Comment (ParryLuke) [reply api.swagger.yaml]: updated

File: api.swagger.yaml [hunk 2]
```diff
@@ -2877,6 +2902,12 @@ components:
             - inactive
             - pending
     group-schema:
+      required:
+        - title
+        - description
+        - group_category
```
Comment (jmorahan) [inline api.swagger.yaml]:
```suggestion
        - group_category_uuid
```
right?

Comment (ParryLuke) [reply api.swagger.yaml]: yes thanks

**Pull request merged**